scripts/bugswarm_filter.py

```diff
@@ -14,16 +14,21 @@
 
 bugswarmapi = DatabaseAPI(token=sys.argv[1])
 
-api_filter =\
-'{"lang":{"$in":["Java"]},' +\ # JAVA source language
-'"stability":"5/5",' +\ # non-flaky
-'"classification.exceptions":["NullPointerException"],' +\ # NPE error type
+api_filter = (
```

Comment:
This is a much nicer way to split a string over several lines than what I had done -- filing it away for future python scripting!